### PR TITLE
Add heimdall to main cluster

### PR DESCRIPTION
**Pull request by Atralupus**
Implement heimdall network

### CODE REVIEW

File: 9c-main/multiplanetary/network/heimdall.yaml [hunk 1]
```diff
@@ -0,0 +1,430 @@
+# Default values for 9c-main cluster.
+# This is a YAML-formatted file.
+
+clusterName: "9c-main-v2"
+logLevel: "debug"
+
+global:
+  image:
+    repository: "longfin/9c-headless"
+    tag: "multiplanetary-2023111801"
```
Comment (area363) [inline 9c-main/multiplanetary/network/heimdall.yaml:10]:
```suggestion
```
Remove this value to use the general image?

Comment (Atralupus) [reply 9c-main/multiplanetary/network/heimdall.yaml:10]: Now, the general's image is the mainnet headless image, but we should set the image to `longfin/9c-headless:multiplanetary-2023111801` only in heimdall

File: 9c-main/multiplanetary/network/heimdall.yaml [hunk 2]
```diff
@@ -0,0 +1,430 @@
...
+
+  loggingEnabled: true
+
+  extraArgs:
+  - --tx-quota-per-signer=1
+  - --consensus-target-block-interval=7000
+  - --config=appsettings.configmap.json
+
+  consensusSeedStrings:
+  - "03380b4ba8722057c9b4d8594f8de9481eb296aba4b3c168666f57b17596452ae7,heimdall-tcp-seed-1.planetarium.dev,31235"
```
Comment (area363) [inline 9c-main/multiplanetary/network/heimdall.yaml:154]:
```suggestion
  - "03380b4ba8722057c9b4d8594f8de9481eb296aba4b3c168666f57b17596452ae7,heimdall-seed-1.nine-chronicles.com,31235"
```

File: 9c-main/multiplanetary/network/heimdall.yaml [hunk 3]
```diff
@@ -0,0 +1,430 @@
...
+  image:
+    repository: planetariumhq/market-service
+    pullPolicy: Always
+    tag: git-bed6517646e0e5b8ba44700406f714b9cf389782
+
+  env:
+  - name: DOTNET_gcServer
+    value: "1"
+  - name: RpcConfig__Host
+    value: heimdall-main-full-state.nine-chronicles.com
```
Comment (area363) [inline 9c-main/multiplanetary/network/heimdall.yaml:253]:
```suggestion
    value: heimdall-full-state.nine-chronicles.com
```

File: 9c-main/multiplanetary/network/heimdall.yaml [hunk 4]
```diff
@@ -0,0 +1,430 @@
...
+  - name: WorkerConfig__SyncShop
+    value: "false"
+  - name: WorkerConfig__SyncProduct
+    value: "false"
+
+  envForWrite:
+  - name: DOTNET_gcServer
+    value: "1"
+  - name: RpcConfig__Host
+    value: heimdall-main-full-state.nine-chronicles.com
```
Comment (area363) [inline 9c-main/multiplanetary/network/heimdall.yaml:265]:
```suggestion
    value: heimdall-full-state.nine-chronicles.com
```

File: 9c-main/multiplanetary/network/heimdall.yaml [hunk 5]
```diff
@@ -0,0 +1,430 @@
...
+      eks.amazonaws.com/nodegroup: heimdall-m7g_xl_2c
+
+fullState:
+  enabled: true
+
+  loggingEnabled: true
+
+  useTurnServer: false
+
+  host: "heimdall-main-full-state.nine-chronicles.com"
```
Comment (area363) [inline 9c-main/multiplanetary/network/heimdall.yaml:286]:
```suggestion
  host: "heimdall-full-state.nine-chronicles.com"
```

File: 9c-main/multiplanetary/network/heimdall.yaml [hunk 6]
```diff
@@ -0,0 +1,430 @@
...
+    node.kubernetes.io/instance-type: m5.large
+
+fullState:
+  enabled: true
+
+  loggingEnabled: true
+
+  useTurnServer: false
+
+  host: "heimdall-main-full-state.nine-chronicles.com"
```
[the remark this answers is not in the document]
Comment (Atralupus) [reply 9c-main/multiplanetary/network/heimdall.yaml]: done

**Pull request merged**